sources/engine/Xenko.Engine/Rendering/Lights/LightClusteredPointSpotGroupRenderer.cs

```diff
@@ -162,7 +162,7 @@ internal class PointLightShaderGroupData : LightShaderGroupDynamic
             private FastListStruct<Int2> clusterInfos = new FastListStruct<Int2>(8);
             private RenderViewInfo[] renderViewInfos;
             private Int2 maxClusterCount;
-            private Plane[] zPlanes;
+            //private Plane[] zPlanes;
```

Comment:
Remove, don't comment out.

Reply:
OK as is, let's delete all related code in a separate pass